data.sql

```diff
@@ -166,3 +166,91 @@ UPDATE animals SET owner_id = 3 WHERE name IN  ('Devimon' ,'Plantmon');
 UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander' ,'Squirtle' ,'Blossom');
 UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon' ,'Boarmon');
 COMMIT;
+
+
+BEGIN;
+INSERT INTO vets (name, age, date_of_graduation) 
+  VALUES ('William Tatcher',  45, '2000-04-23'),
+    ('Maisy Smith', 26, '2019-01-17'),
+    ('Stephanie Mendez', 64, '1981-05-04'),
+    ('Jack Harkness', 38, '2008-06-08');
+END;
```

Comment:
- [ ] Let us modify our insertion using this way, to make our queries more readable:

  `INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('William Tatcher', 45, '2000-04-23');`

   `INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('Maisy Smith', 26, '2019-01-17');`

   `INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('Stephanie Mendez', 64, '2019-05-17');`

   `INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('Jack Harknes', 38, '2008-06-08');`